src/components/Sidebar.js

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect, useState } from "react";
+import { Link, Outlet } from 'react-router-dom';
+import { styled, useTheme } from '@mui/material/styles';
+import { getCollections } from '../common/client';
+import { Divider, Drawer, FormControl, TextField, Grid, IconButton, List, ListItem, ListItemIcon, ListItemText } from "@mui/material";
+import { Logo } from "../components/Logo";
+import { Stack, Button } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import { Typography } from '@mui/material';
+import Dataset from "@mui/icons-material/Dataset";
+import { purple } from '@mui/material/colors';
+
+
+const StyledLink = styled(Link)`
+  color: black;
+  text-decoration: none;
+  margin: 1rem;
+  position: relative;
+`;
```

Comment:
I would prefer as little custom styles as possible. Instead of creating `StyledLink` we can use `ListItemButton`, which does everything out of the box